lib/mais/client.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'faraday/httpclient'
+require 'faraday/retry'
```

Comment:
specs passed without this, presumably due to some rails or app magic, but I included it to be clearer (?)